diff --git a/bin/fake_edit b/bin/fake_edit
index e5a1d76..0db1160 100755
--- a/bin/fake_edit
+++ b/bin/fake_edit
@@ -3,4 +3,5 @@
 if ! tty -s <&1; then
   exec >/dev/null
 fi
+test -z "$1" && echo>&2 "cannot open '$1'" && exit 1;
 echo "No log message" | tee "$1" 
diff --git a/bin/rsync_sync b/bin/rsync_sync
old mode 100644
new mode 100755
index 028363c..58915f2
--- a/bin/rsync_sync
+++ b/bin/rsync_sync
@@ -1,6 +1,7 @@
 rsync_sync () 
 {
   while(($#)); do
+    ssh $1 mkdir -p "$PWD"
     rsync -Parv $1 :$PWD/. . && rsync -Parv . $1 :$PWD/.
     shift;
   done
diff --git a/bin/tty_say b/bin/tty_say
old mode 100755
new mode 100644
